opcUaDevSupApp/devOpcUa.c

```diff
@@ -276,10 +276,13 @@ long init_longin (struct longinRecord* prec)
 
 long read_longin (struct longinRecord* prec)
 {
+    OPCUA_ItemINFO* pOPCUA_ItemINFO = NULL;
```

Comment:
+1
@bkuner: While this is allowed in C++ and C99, some C compilers do not accept variable declarations in the middle of the code, only at the beginning of a block. (I.e. after any opening `{` bracket.)